rest-backend/src/main/java/org/eclipse/steady/backend/repo/LibraryIdRepository.java

```diff
@@ -38,9 +38,6 @@
   /** Constant <code>FILTER</code> */
   public static final ResultSetFilter<LibraryId> FILTER = new ResultSetFilter<LibraryId>();
 
-  @Query("SELECT l FROM LibraryId l WHERE l.id=:id")
```

Comment:
Very nice that those are not necessary any more, but how comes?

Reply:
As far as i remember, spring 2.x introduced some changes to the CRUDRepository (some details in the spring 1.5.x to 2.x migration). In particular findOne was renamed to findById, as a result we do not need to define findById on our own and we now use the one offered by the CRUDRepository.